src/tabs/setup.html

```diff
@@ -226,21 +226,24 @@
                                     <td id="api-version" i18n="initialSetupInfoAPIversion"></td>
                                     <td class="api-version"></td>
                                 </tr>
-                                    <tr>
+                                <tr>
                                     <td id="build-date" i18n="initialSetupInfoBuildDate"></td>
                                     <td class="build-date"></td>
                                 </tr>
                                 <tr>
                                     <td id="build-type" i18n="initialSetupInfoBuildType"></td>
                                     <td class="build-type"></td>
                                 </tr>
-                                </tr>
```

Comment:
Two of the `<tr>` were over-indented, but THIS `</tr>` seems to have been EXTRA so I removed it.